src/picongpu/include/particles/ionization/byField/BSIEffectiveZ/AlgorithmBSIEffectiveZ.hpp

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2014-2016 Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "pmacc_types.hpp"
+#include "simulation_defines.hpp"
+#include "particles/traits/GetIonizationEnergies.hpp"
+#include "particles/traits/GetAtomicNumbers.hpp"
+#include "particles/traits/GetEffectiveAtomicNumbers.hpp"
+#include "traits/attribute/GetChargeState.hpp"
+#include "algorithms/math/floatMath/floatingPoint.tpp"
```

Comment:
already included in `libPMacc/include/algorithms/math.hpp` which is included in PIConGPU's `simulation_types.hpp` which is already present in PIConGPU's `simulation_defines.hpp`